### DISCUSSION TITLE
covariate for CL

### DISCUSSION THREAD

**Discussion by Mannny-PK** [LAPKB/Pmetrics#109] [Q&A] [answered]
Dear Professor,
I have encountered a problem when I used Pmetrics, the details are as follows.
The code to run is as follows:
#Primary
CL0, 0.1,3
Q, 0.1,6
V, 1,60
Vp, 1,120

#Sec

Ke = CL/V

KCP = Q/V

KPC = Q/Vp

CL = CL0-0.05*ALB

C 32 is the mean alb  in the population

#Cov

WT

CCR

ALB

AGE

APACHE

#Out
Y(1) = X(1)/V

#Err
G=5
0.1, 0.15, 0, 0

The run was failed.
![图片1](https://user-images.githubusercontent.com/109020077/199678286-6b3cbbb5-6463-41ef-bea4-2fafa2de146b.png)

**Comment by mnneely** [comment, marked as answer]
You might want to try moving the equations around in the #SEC block, so that CL is defined before it is used, i.e. make it the first equation. Not sure if that is the problem, but I’d start there.  I also usually put the #COV block before the #SEC block if I’m using covariates in equations, although that should not be an issue. Unfortunately, I can’t read the Chinese characters in the screenshot, so I don’t know what it says.

From: Mannny-PK ***@***.***>
Date: Thursday, November 3, 2022 at 1:43 AM
To: LAPKB/Pmetrics ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [LAPKB/Pmetrics] covariate for CL (Discussion #109)

Dear Professor,
I have encountered a problem when I used Pmetrics, the details are as follows.
The code to run is as follows:
#Primary
CL0, 0.1,3
Q, 0.1,6
V, 1,60
Vp, 1,120

#Sec

Ke = CL/V

KCP = Q/V

KPC = Q/Vp

CL = CL0-0.05*ALB

C 32 is the mean alb in the population

#Cov

WT

CCR

ALB

AGE

APACHE

#Out
Y(1) = X(1)/V

#Err
G=5
0.1, 0.15, 0, 0

The run was failed.
[??1]<https://urldefense.com/v3/__https:/user-images.githubusercontent.com/109020077/199678286-6b3cbbb5-6463-41ef-bea4-2fafa2de146b.png__;!!LIr3w8kk_Xxm!uTEp7Map9DyoK9w7l6I3A5efdtof2b9Qs639uxK26a_X4Z6992Yn4QIYun7xw-U9fql-pVuAmmeStlB6kvrENQo$>

**Reply by Mannny-PK** [reply]
I reworked it based on your suggestion and it worked, thank you very much for your answer!
